pkg/kudoctl/util/repo/repo.go

```diff
@@ -0,0 +1,42 @@
+package repo
+
+// RepositoryConfiguration represents a collection of parameters for framework repository
+type RepositoryConfiguration struct {
+	//LocalPath string `json:"localPath"`
+	URL string `json:"url"`
+}
+
+// Default returns a given RepositoryConfiguration struct
+func Default() *RepositoryConfiguration {
```

Comment:
This can be a variable, like such:

```go
var DefaultRepo *RepositoryConfiguration = &RepositoryConfiguration{
    URL: "https://kudo-repository.storage.googleapis.com",
}
```

This'll prevent a function invocation and allows the Go compiler to potentially perform inlining optimizations.